overview.qmd

```diff
@@ -45,15 +45,20 @@ text <- c(
   "  \"polars\",",
   "  repos = c(",
   "    \"https://production.r-multiverse.org\",",
-  sprintf("    \"https://packagemanager.posit.co/cran/%s\"", snapshot_date()),
+  sprintf("    \"https://packagemanager.posit.co/cran/%s\"", staging_date()),
   "  )",
   ")",
   "```"
 )
 cat(text, sep = "\n")
 ```
 
-[Production](production.qmd) is deployed in snapshots, and it works best with compatible versions of base R and dependency packages.
+[Production](production.qmd) is deployed in snapshots.
+The current snapshot works best with:
+
+1. The version of R-release from the date of the snapshot (`r snapshot_date()`).
+1. CRAN dependencies from the first day of the [Staging period](production.qmd#staging) (`r staging_date()`).
```

Comment:
not intuitive :sweat_smile: 

btw for things like that we might want to also have a blog with deep dives.

Reply:
maybe link to production page since it has a good explanation?

Reply:
> not intuitive

It's a tough one to explain concisely

> maybe link to production page since it has a good explanation?

Is `[Staging period](production.qmd#staging)` sufficient?

Reply:
yes